scripts/lib/CIME/code_checker.py

```diff
@@ -92,7 +95,8 @@ def check_code(files, num_procs=10, interactive=False):
     # is a valid file, if not, we search the repo for a file with similar name.
     files_to_check = []
     if files:
-        repo_files = run_cmd_no_fail('git ls-files', from_dir=get_cime_root(), verbose=False).splitlines()
+        repo_files = get_all_checkable_files()
+#        repo_files = run_cmd_no_fail('git ls-files', from_dir=get_cime_root(), verbose=False).splitlines()
```

Comment:
@jgfouca looking for feedback on this change - if it's okay I'll remove the commented code.

Reply:
It looks ok. I can't remember why I used ls-files instead of checkable_files.